src/active-response/netsh.c

```diff
@@ -12,14 +12,60 @@
 #include "active_responses.h"
 
 #define RULE_NAME "WAZUH ACTIVE RESPONSE BLOCKED IP"
-#define NETSH "C:\\Windows\\System32\\netsh.exe"
+#define NETSH     "C:\\Windows\\System32\\netsh.exe"
+#define REG       "C:\\Windows\\System32\\reg.exe"
+
+#define PATH_FIREWALL_PROFILES_REG_DEFAULT "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\SharedAccess\\Parameters\\FirewallPolicy\\"
+#define FIREWALL_DATA_INITIALIZE {false, false, FIREWALL_DOMAIN}
+#define FIREWALL_PROFILES_MAX (3)   /*!< Maximun number of profiles*/
+
+typedef struct {
+    char *log_msg;
+    cJSON *input_json;
```

Comment:
I think you don't need this. Free the JSON here https://github.com/wazuh/wazuh/pull/14804/files#diff-20cc756527cbb2a9d85240db689dd920417d7d2c629041e7f6c9e05ddfe0778aR120